tests/topotests/bgp_vrf_lite_ipv6_rtadv/test_bgp_vrf_lite_ipv6_rtadv.py

```diff
@@ -68,6 +69,19 @@ def setup_module(mod):
     router_list = tgen.routers()
 
     logger.info('Testing with VRF Lite support')
+    krel = platform.release()
+
+    # May need to adjust handling of vrf traffic depending on kernel version
+    l3mdev_accept = 0
+    if topotest.version_cmp(krel, '4.15') >= 0 and \
+       topotest.version_cmp(krel, '4.18') <= 0:
```

Comment:
4.19 skipped?

Reply:
we've got this test in several places, to work around an issue in a range of kernels. I'm poking at this because I see something similar with the new 5.x kernels, which is a bit of a surprise.